Cargo.toml

```diff
@@ -95,12 +95,13 @@ ark-bls12-381 = { git = "https://github.com/arkworks-rs/curves/", rev = "3fded1f
 [profile.release]
 codegen-units = 1
 lto = true
+opt-level = 3
 
 [profile.release-unoptimized]
 inherits = "release"
 
 [profile.release-unoptimized.package.example]
-opt-level = 0
+opt-level = 3
```

Comment:
This profile is called `unoptimized`. It's confusing to have it optimized. I think we should rename the profile or leave the optimization level at zero.

Reply:
We should just remove `release-unoptimized` from the codebase and replace it with `release`.

This won't automatically update projects people have already made using `cargo nexus new` or `cargo nexus host`, but is much better for the future.